### PR TITLE
Grammatically updated readme

### PR DESCRIPTION
**Pull request by aprakash07**
Fixes #

#### Short description of what this resolves:
It updates few minor grammatical errors.

#### Changes proposed in this pull request:
Under Contributing section in line 13 changes were made-
OpenEMR is a leader in healthcare open source software and comprises a large and diverse community of software developers, medical providers, and educators with a healthy mix of both our volunteers and professionals. 

changes:
OpenEMR is a leader in healthcare open source softwares and comprises 'of' a large and diverse community of software developers, medical providers and educators with an 'effective blend' of both our volunteers and professionals. 

Under Reporting issues and bugs in line 25
If you are unsure if it is an issue/bug, then always feel free to use the

changes
If you are unsure 'whether' it is an issue 'or a' bug, then always feel free to use the

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 ### Contributing
 
-OpenEMR is a leader in healthcare open source software and comprises a large and diverse community of software developers, medical providers and educators with a very healthy mix of both volunteers and professionals.  [Join us and learn how to start contributing today!](https://open-emr.org/wiki/index.php/FAQ#How_do_I_begin_to_volunteer_for_the_OpenEMR_project.3F)
+OpenEMR is a leader in healthcare open source softwares and comprises of a large and diverse community of software developers, medical providers and educators with an effective blend of both our volunteers and professionals.  [Join us and learn how to start contributing today!](https://open-emr.org/wiki/index.php/FAQ#How_do_I_begin_to_volunteer_for_the_OpenEMR_project.3F)
```
Comment (stephenwaite) [inline README.md:13]: hi @aprakash07 , would leave it as `software and comprises a...`

how about `an effective blend of volunteers and professionals.`?

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Extensive documentation and forums can be found on the [OpenEMR website](https:/
 
 ### Reporting Issues and Bugs
 
-Report these on the [Issue Tracker](https://github.com/openemr/openemr/issues). If you are unsure if it is an issue/bug, then always feel free to use the [Forum](https://community.open-emr.org/) and [Chat](https://www.open-emr.org/chat/).
+Report these on the [Issue Tracker](https://github.com/openemr/openemr/issues). If you are unsure whether it is an issue or a bug, then always feel free to use the [Forum](https://community.open-emr.org/) and [Chat](https://www.open-emr.org/chat/).
```
Comment (stephenwaite) [inline README.md:25]: this looks nice, thank you

File: README.md (hunk 1 above)
Comment (sunsetsystems) [reply README.md:13]: "Softwares" is not a word, as "software" already refers to a _collection_ of programs, instructions etc.

"Comprise" is like "contain" or "include", so we don't say "comprises of".

"Both" and "our" seem superfluous here. I'd just say "an effective blend of volunteers and professionals".

Comment (aprakash07) [reply README.md:13]: yes, an effective blend of volunteers and professionals looks fine.

**Pull request closed without merging**